packages/@atjson/source-gdocs-paste/src/text-styles.ts

```diff
@@ -24,7 +25,7 @@ export default function extractTextStyles(
           "-gdocs-va": style.ts_va,
         },
         start: i,
-        end: -1,
+        end: content.length,
```

Comment:
Yeah I think this is definitely the more correct behavior now. Thanks for tracking this down!